src/elmExecutor/buildPopulatedResourceBundle.js

```diff
@@ -1,18 +1,27 @@
 function doSearch(smart, type, fhirVersion, request, callback) {
   const q = {};
-  
+
   // setup the query for Practitioner and Coverage
   switch (type) {
     case "Practitioner":
-      q._id = (request.performer && request.performer.reference);
+      let performer;
+      if (request.resourceType === "DeviceRequest") {
+        performer = request.performer && request.performer.reference;
+      } else if (request.resourceType === "ServiceRequest") {
+        performer = request.performer[0].reference;
```

Comment:
Need to check to make sure that there is at least one performer before grabbing the reference.
performer = request.performer[0] && request.performer[0].reference;